tests/common/nebula_service.py

```diff
@@ -321,6 +321,7 @@ def _make_sa_params(self, **kwargs):
         self.graphd_param['password_lock_time_in_secs'] = '10'
         self.graphd_param['raft_heartbeat_interval_secs'] = '30'
         self.graphd_param['skip_wait_in_rate_limiter'] = 'true'
+        self.graphd_param['add_local_host'] = 'false'        
```

Comment:
UT has already call add hosts explicitly?

Reply:
Ent version hardcoded "add hosts to new zone default_zone", and in nebula repo "add hosts",  which will affect tck.  So I let "add hosts" done by tck start cluster scripts use "--add_local_host".